src/main/java/seedu/address/logic/commands/task/MarkTaskCommand.java

```diff
@@ -0,0 +1,69 @@
+package seedu.address.logic.commands.task;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.TaskCommand;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+
+/**
+ * Marks a task of a person as complete or incomplete.
+ */
+public class MarkTaskCommand extends TaskCommand {
+
+    public static final String COMMAND_WORD = "mark";
+
+    public static final String COMMAND_WORD_FULL = TaskCommand.COMMAND_WORD + " " + COMMAND_WORD;
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD_FULL + ": Marks a task to be complete "
```

Comment:
A little confusing to read, maybe you could consider following the format of DeleteCommand.MESSAGE_USAGE!

Reply:
Noted, I will refine this part.

Reply:
Finished.